File: sentToCK.cpp
#include <iostream>
int main()
{
    using namespace std;
    int msg = 1, sent = 0;
    for (int day = 1; day <= 30; day++) {
        cout << "[Day" << day << "] Messages: " << msg <<endl;
        sent += msg;
        msg *= 2;
    }
    cout << "Total sent: " << sent <<endl;
    return 0;
}

File: sentToCK.cs
using System;

namespace sentToCK
{
    class sentToCK
    {
        static void Main(string[] args)
        {
            int msg = 1;
            int send = 0;
            for(int day = 1; day<=30; day++)
            {
                Console.WriteLine(String.Format("[ Day {0} ] Message: {1}", day, msg));
                send += msg;
                msg *= 2;
            }
            Console.WriteLine("Total sent: " + send);
            Console.ReadLine();
        }
    }
}
